citizenvoice/survey_design/admin.py

```diff
@@ -1,12 +1,3 @@
-from apiapp.models import Answer, Question, Response, Survey
-
 from django.contrib import admin
-import django.contrib.auth.models
-from django.contrib import auth
-
-# model_list = [Answer, Question, Response, Survey]  # iterable list of models
-# admin.site.register(model_list)
```

Comment:
Would it not be handy to still have the other models registered in the admin site here?

Reply:
These are the models that are not used but I think you and Gerdus discussed should not be removed. I think it might be confusing to have these included in the admin site, especially since the names are the same as well.

Reply:
Right, we can just leave them there too keep the general structure of the Django app